src/MultiOwnable.sol

```diff
@@ -1,124 +1,217 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.4;
 
+/// @notice Storage layout used by this contract.
+///
 /// @custom:storage-location erc7201:coinbase.storage.MultiOwnable
 struct MultiOwnableStorage {
-    /// @dev tracks the index of the next owner added
+    /// @dev Tracks the index of the next owner to add.
     uint256 nextOwnerIndex;
-    /// @dev Allows an owner to be idenfitied by a uint256.
-    /// Some uses--such as signature validation for secp256r1 public
-    /// key owners--requires the caller to assert which owner signed.
-    /// To economize calldata, we allow an index to identify an owner,
-    /// so that the full owner bytes do not need to be passed.
-    /// Note, we use uint256 rather than a smaller uint because it
-    /// provides flexibility at little to no cost.
-    /// uint256 allows that we will (practically) never run out of owner indexes.
-    /// And on L2, where calldata gas is a concern,
-    /// we should not be charged for the extra 0 bytes.
-    mapping(uint256 => bytes) ownerAtIndex;
-    mapping(bytes => bool) isOwner;
+    /// @dev Mapping of indices to raw owner bytes, used to idenfitied owners by their
+    ///      uint256 id.
+    ///
+    ///      Some uses —-such as signature validation for secp256r1 public key owners—-
+    ///      requires the caller to assert which owner signed. To economize calldata,
+    ///      we allow an index to identify an owner, so that the full owner bytes do
+    ///      not need to be passed.
+    ///
+    ///      The underlying raw bytes can either be:
+    ///         - An abi encoded ethereum address
+    ///         - The abi encoded public key (x, y) coordinates when using passkey.
+    mapping(uint256 index => bytes owner) ownerAtIndex;
+    /// @dev Mapping of raw bytes accounts to booleans indicating whether or not the
+    ///      account is an owner of this contract.
+    mapping(bytes account => bool isOwner_) isOwner;
 }
 
 /// @title Multi Ownable
+///
 /// @notice Auth contract allowing multiple owners, each identified as bytes.
+///
 /// @author Coinbase (https://github.com/coinbase/smart-wallet)
 contract MultiOwnable {
-    /// keccak256(abi.encode(uint256(keccak256("coinbase.storage.MultiOwnable")) - 1)) & ~bytes32(uint256(0xff))
-    bytes32 private constant MultiOwnableStorageLocation =
+    /// @dev Slot for the `MultiOwnableStorage` struct in storage.
+    ///      Computed from: keccak256(abi.encode(uint256(keccak256("coinbase.storage.MultiOwnable")) - 1)) & ~bytes32(uint256(0xff))
+    ///      Follows ERC-7201 (see https://eips.ethereum.org/EIPS/eip-7201).
+    bytes32 private constant MUTLI_OWNABLE_STORAGE_LOCATION =
         0x97e2c6aad4ce5d562ebfaa00db6b9e0fb66ea5d8162ed5b243f51a2e03086f00;
 
+    /// @notice Reverted when the sender is not an owner and is trying to call a privileged function.
```

Comment:
Picking up where the comments ended on the previous pr...

This reads weird to me still. The errors aren't reverting; they are being  thrown when reverting. I won't die on this hill but i'm not a fan.

Reply:
Yeah I think "thrown when" might be better phrasing :)

Reply:
Still thinking about this, given the call is `revert error()` rather than `throw error()` maybe @xenoliss had this right 🤔 

Although the [docs use thrown](https://docs.soliditylang.org/en/v0.8.24/contracts.html#function-modifiers) so let's stick with it

Reply:
I'm used to `Reverted when XXX` so I am defiitely biased but when re-reading the docs it is true that errors are not reverted; the state is.